gh-actions/go/gotestfmt/gotestfmt

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# This script is a wrapper around gotestfmt. It expects the output of `go test -json`
+# on stdin. The output of failed tests is written to stdout, while the full output
+# (including successful tests) is written to a logfile.
+
+usage(){
+    echo "Usage: $0 [--logfile <logfile>] [-h|--help]"
+}
+
+# Parse options
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --logfile)
+            LOGFILE="$2"
+            shift 2
+            ;;
+        -h|--help)
+            usage
+            exit 0
+            ;;
+        *)
+            echo "Unknown option: $1"
+            usage
+            exit 1
+            ;;
+    esac
+done
+
+LOGFILE="${LOGFILE:-$(mktemp -t gotestfmt.XXXXXX.log)}"
+echo >&2 "gotestfmt: Logging to $LOGFILE"
+STDERR_FILE=${LOGFILE%.log}.stderr
```

Comment:
FYI it's not needed for shell assignments (there's no risk for spaces of being skipped)

Reply:
Right, but we had this discussion with @adombeck on the automatic tagging/versionning snap and we added them even when they were not useful, to have a single style, being `"${FOO}"` for all variable reads. I think we should follow this single style here.

Reply:
I don't remember that we decided that. We should always use quotes to avoid word splitting, but as @3v1n0 said, in variable assignments there is no word splitting, even if the variable contains spaces.

Reply:
right! fixed :)